html/read_config.php

```diff
@@ -17,7 +17,6 @@
 $config = new Config_Lite('user_config.ini');
 $standalone = '';
 $gmail = '';
-$cloud = '';
```

Comment:
Why to remove $cloud initialization? I believe I was having an error when the variable wasn't initialized.